addons/account/models/chart_template.py

```diff
@@ -265,8 +265,8 @@ def _pre_reload_data(self, company, template_data, data, force_create=True):
                 template_data.pop(prop)
         data.pop('account.reconcile.model', None)
         if 'res.company' in data:
+            data['res.company'][company.id].clear()
             data['res.company'][company.id].setdefault('anglo_saxon_accounting', company.anglo_saxon_accounting)
```

Comment:
Why did you keep it?

Reply:
that part was added to prevent of setting anglo_saxon_accounting to false

Reply:
https://github.com/odoo/odoo/commit/1f26e5e849f3b27485d8a3f802fe2b3b74ba91c9

that change was done here

Reply:
I know, but if the value is not in the dict, we won't change it to False...
Clearing or popping the dict leads to the same value, without the useless write (+ triggers etc)

Reply:
No it will reset to False here 
![image](https://github.com/user-attachments/assets/c06a9a4e-4c18-4008-80d7-f393b2fd01ad)

I do not know the impact of removing this line so I decided to keep it